### PR TITLE
Adds Winter coats to cargo orders

### PR DESCRIPTION
**Pull request by robosperg**
Probably powercreep in some way
Draft until Otatoh finishes some more coats
Kind of threw stuff around wherever, can rearrange or add locks to crates if wanted, also need to make a pack for winter boots or just bundle them with crates, undecided on which way to go about it.
:cl:
 * rscadd: Winter coats are now available at cargo, expect more in the future

### CODE REVIEW

**Comment by Kharkov72** [comment]
make it a vending machine coward

**Comment by robosperg** [comment]
gimme sprites and I will

File: code/datums/supplypacks.dm [hunk 1]
```diff
@@ -728,6 +728,110 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 	cost = 150
 	containertype = /obj/structure/closet/crate/basic
 	containername = "contacts crate"
+	group = "Clothing"	
+	
+//Winter Coats//
+	
+	/datum/supply_packs/engwinter
```
Comment (ShiftyRail) [inline code/datums/supplypacks.dm]: All of this is indented for some reason
It shouldn't be
Move everything 1tab backwards

**Comment by robosperg** [comment]
maybe fixed

**Review by ShiftyRail — APPROVED** [review]

**Comment by robosperg** [comment]
gonna call this ready as-is, can just add coats as they come

**Pull request merged**